web/js/layoutManager.js

```diff
@@ -50,6 +86,9 @@ LayoutViewport, ItemsHandler */
     Utils.addEventsHandlers('layoutMenuView:', handlers, global);
     Utils.addEventsHandlers('layoutView:', handlers, global);
     Utils.addEventsHandlers('hangout:', handlers, global);
+    window.addEventListener("orientationchange", throttledAdjustLayout, false);
```

Comment:
```suggestion
    window.addEventListener('orientationchange', throttledAdjustLayout);
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

this article mentions that useCapture is not optional to enhance the browser compatibility.

Reply:
This is an outdated file. the latest commit doesn't have this code.